### PR TITLE
Adds welcome sounds for traitors and traitor silicons

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
soundadd: Syndicate agents, double agents, hacked cyborgs, nuclear operatives, traitor AI's, and syndicate cyborgs have two cool welcome messages.
/:cl:

These are cool sounds and I'm tired of missing my traitor rounds. Bay's licensing is weird but hey its compatible.

To hear the sounds click the first commit and then view.

### CODE REVIEW

**Comment by Armhulen** [comment]
The AI voiceline is not fucking cannon

**Comment by Incoming5643** [comment]
if a license can be safely contained to a file in a folder it should be contained to a file in a folder. 

Readme bloat is real.

**Comment by Armhulen** [comment]
that too i guess

**Comment by Bawhoppen** [comment]
can you transcribe them for me my headphones are broken

**Comment by Incoming5643** [comment]
totally subjective but I find them a little tacky

**Comment by Bawhoppen** [comment]
whatever incoming says I agree with

**Comment by MrPerson** [comment]
Travis pls wtf is wrong with you

**Comment by Armhulen** [comment]
I don't like them, they would not send syndicates to one of the most dangerous stations FOR THEIR FIRST MISSION

**Comment by Armhulen** [comment]
and the voice is not garbled enough these are greedy fucks not the guy who takes my mcdonalds order

**Comment by Supermichael777** [comment]
The generic one is:

>"Mission specifications decrypted, welcome to the syndicate"

The ai and cyborg emag one is:

> "Alert: syndicate hacking attempt in progress, core security breach will/ This intelligence is now property of the syndicate; your new directives have been uploaded."

@Armhulen SECURITY BRIEFING, LEVEL 3: Syndicate agents are specifically sleeper agents with no memory of ever being a member before or knowledge of previous missions. To ensure long term goals cannot be discovered or disclosed all memories of missions are apparently wiped by their handlers once they are picked up.  Any agent who does not report to a handler is remotely wiped by an unknown method. All captured strike team operatives have had their implanted bomb remotely detonated; this seems to be a common punishment for failure.  Deny the existence of any hostile activity to crew below clearance level 3. This restriction may be removed by a level 2 or above officer or by a red alert status.  Don't let the leak be you, report secure document leakage immediately!

**Comment by Armhulen** [comment]
"Alert: syndicate hacking attempt in progress, core security breach will/ This intelligence is now property of the syndicate; your new directives have been uploaded."

still triggered

**Comment by RandomMarine** [comment]
I like the idea of an audio notification of traitor status but I don't really like those sounds in particular.

**Comment by Niknakflak** [comment]
Oh god I love these sound files.  It gives me such a mission impossible 1 evil villain vibe

**Comment by ma44** [comment]
*BEEP* VALID HERE

**Comment by Cyberboss** [comment]
Nice files, but *deep lore* states that the AI is malfunctioning, not part of the syndicate.

**Comment by WJohn** [comment]
Yeah I'm not a fan of these, especially since I know that @cuboos made actual intro sounds for LOTS of different traitor roles that never got implemented for some reason.

**Comment by KorPhaeron** [comment]
Decryption doesnt fit my head canon

**Comment by Cyberboss** [comment]
Aren't traitors sleeper agents? You should do a clip from the CoD:BO numbers woman

**Comment by Supermichael777** [comment]
@Cyberboss licensing memes

Everyone else if you have better sounds that we can use please post

**Comment by Cuboos** [comment]
https://www.dropbox.com/s/skhr8zs829vm6w4/tater_alert_mixdown.wav?dl=0

was this the one you were talking about?

**Comment by PKPenguin321** [comment]
Yeah I was about to say @Cuboos did this a long time ago with a lot more sounds that were really cool but it didn't go through for some reason, I should dig up the PR

**Comment by PKPenguin321** [comment]
AHA
https://github.com/tgstation/tgstation/pull/6301

**Comment by Cuboos** [comment]
Well, if enough people want this i can restart the project. I just finished up my last class for college, so i should have a little more free time now. 

ALSO https://www.dropbox.com/s/zx9guuxi1xo35yy/tater_alert_mixdown2.wav?dl=0

i just remembered, this was the final version of the alert.

**Comment by WJohn** [comment]
that final version could easily do with the last 4 seconds of silence being cut off to save space.

**Comment by Cuboos** [comment]
This was made a while ago before i figured out how to fix that.

**Comment by Niknakflak** [comment]
I like the bay ones though

**Comment by Cuboos** [comment]
Bay ones?

**Comment by PKPenguin321** [comment]
give me cuboos sounds or give me death

**Comment by Supermichael777** [comment]
those aren't quite overt enough, sound too much like existing ambiance.

**Comment by Cuboos** [comment]
What, my sounds or Bay's?

**Comment by PKPenguin321** [comment]
GIVE cuboos sounds

**Comment by Niknakflak** [comment]
yea no, bay sounds > cuboos sounds

**Comment by Bawhoppen** [comment]
no sounds > any sounds

**Comment by Cuboos** [comment]
I have no idea what's going on in this thread anymore...

**Comment by Bawhoppen** [comment]
CLOSE AND START FRESH

**Comment by PKPenguin321** [comment]
GIVE ME CUBOOS SOUNDS **NOW NOW NOW**

**Comment by Niknakflak** [comment]
Bayyyyy sounds

**Comment by PKPenguin321** [comment]
REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Comment by Cuboos** [comment]
can i get a link to these bay sounds?

**Comment by PKPenguin321** [comment]
>To hear the sounds click the first commit and then view.

They're neat but they're not cuboos sounds

**Comment by Cuboos** [comment]
is there a way to do it without downloading them?

**Comment by Cuboos** [comment]
These are awful... where the layering? Where's the flair? Where's the effort? If you want voices i got a better TTS program that the free ATT one.

File: code/game/gamemodes/traitor/traitor.dm [hunk 1]
```diff
@@ -203,8 +203,10 @@
 /datum/game_mode/proc/finalize_traitor(var/datum/mind/traitor)
 	if(issilicon(traitor.current))
 		add_law_zero(traitor.current)
+		traitor.current << sound('sound/voice/AISyndiHack.ogg')
 	else
 		equip_traitor(traitor.current)
+		traitor.current << sound('sound/voice/syndicate_intro.ogg')
```
Comment (Cyberboss) [inline code/game/gamemodes/traitor/traitor.dm]: use `current.playsound_direct(sound)`

File: code/modules/mob/living/silicon/robot/robot_defense.dm [hunk 2]
```diff
@@ -147,6 +147,7 @@
 	to_chat(src, "<span class='danger'>Initiating diagnostics...</span>")
 	sleep(20)
 	to_chat(src, "<span class='danger'>SynBorg v1.7 loaded.</span>")
+	src << sound('sound/voice/AISyndiHack.ogg')
```
Comment (Cyberboss) [inline code/modules/mob/living/silicon/robot/robot_defense.dm]: `playsound_direct`

**Pull request closed without merging**